perllib/FixMyStreet/Cobrand/Hampshire.pm

```diff
@@ -0,0 +1,38 @@
+package FixMyStreet::Cobrand::Hampshire;
+use parent 'FixMyStreet::Cobrand::Whitelabel';
+
+use strict;
+use warnings;
+
+use Moo;
+with 'FixMyStreet::Roles::ConfirmOpen311';
+with 'FixMyStreet::Roles::ConfirmValidation';
+
+sub council_area_id { return 2227; } # https://mapit.mysociety.org/area/2227.html
+sub council_area { return 'Hampshire'; }
+sub council_name { return 'Hampshire County Council'; }
+sub council_url { return 'hampshire'; }
+sub base_url { return FixMyStreet->config('BASE_URL'); }
+
+# Created with bin/fixmystreet.com/generate_council_location
+sub disambiguate_location {
```

Comment:
There is no front end cobrand, so this function isn't necessary. But no harm in having it for possible future, I guess (apart from thinking it was used).